examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>Are you sure this is a good source? It has not citations and it says</p><p> </p><blockquote>Navigation satellites mainly operate in Medium Earth Orbit (MEO), Geostationary Earth Orbit (GEO), and Inclined Geosynchronous Satellite Orbit (IGSO). These orbits lie in or repeatedly cross the outer Van Allen belt of Earth. With large uncertainty, the radiation in these orbits mainly comes from electrons trapped in the outer radiation belt and solar energetic protons</blockquote><p>This is a different orbit and in S2 orbit you don't go through van allen belts, I think?</p><p> </p><p>I am also a bit confused about that integral. You assume constant, linear dose, why do you need H(x) at all then?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>As you can see there is another paper cited in the notebook: <a href="https://nepp.nasa.gov/files/28265/NEPP-CP-2016-Xapsos-Paper-TNS-TN35529.pdf" target="_blank">Inclusion of Radiation Environment Variability in Total Dose Hardness Assurance Methodology (nasa.gov)</a></p><p> </p><p>I forgot to cite it also when reporting the formula (I will fix this). This one has around 30 citations and provide a full explanation to the formula. The idea is that predicting the dose is not easy and there is some likelihood the dose is higher than expected. So, the idea is to model everything probabilistically so 1 - H(x) provides the probability that the dose is higher than x. </p><p> </p><p>In that paper they use the same formula for a sun-synchronous LEO orbit (Landsat-8) at 700 km altitude (similar inclination to S2).</p>

Reply:
<p>Can you just add a sentence like this to the cell too, so it is clear for other people who will read it </p><p> </p><blockquote>The idea is that predicting the dose is not easy and there is some likelihood the dose is higher than expected. So, the idea is to model everything probabilistically so 1 - H(x) provides the probability that the dose is higher than x. </blockquote>